onnxruntime/core/providers/cuda/cuda_execution_provider_info.h

```diff
@@ -54,6 +54,9 @@ struct CUDAExecutionProviderInfo {
 
   bool enable_cuda_graph{false};
 
+  // By default, for Conv1D, will pad [N,C,D] to [N,C,D,1], if turn on, will pad to [N,C,1,D].
```

Comment:
Is there a way to know which option is best?

Reply:
Not an export of this. PR #7348 and #7702 have discussion on this. Since there was regression for inference, currently still keep it turn off by default. Looks this is model specific.